dist/dfu.js

```diff
@@ -253,103 +309,110 @@
             function handleControl(event) {
                 var data = event.target.value;
                 var view = new DataView(data);
+                
                 var opCode = view.getUint8(0);
+                var req_opcode = view.getUint8(1);
+                var resp_code = view.getUint8(2);
```

Comment:
It's my understanding you shouldn't grab the req_opcode and resp_code here as the opCode hasn't been tested and could be '17', meaning the uint8 reads could be gibberish. Is it a problem?